package.json

```diff
@@ -45,6 +45,12 @@
 					"id": "docsView.documentation",
 					"name": "Documentation",
 					"contextualTitle": "Documentation"
+				},
+				{
+					"type": "webview",
```

Comment:
Since this a new view not everyone may want, I'd set `"visibility": "collapsed"` as the default

Reply:
Agree. Will make that change.